components/recruitment/index.tsx

```diff
@@ -1,102 +1,170 @@
-import { RecruitmentProps } from '@interfaces/recruitment';
-import Landing from '@components/recruitment/Landing';
-import WhoShouldJoin from '@components/recruitment/WhoShouldJoin';
+// import { RecruitmentProps } from '@interfaces/recruitment';
+// import Landing from '@components/recruitment/Landing';
+// import WhoShouldJoin from '@components/recruitment/WhoShouldJoin';
```

Comment:
If we don't use this anymore, let's delete the imports and delete the files as well

Reply:
Done!